docs/user-guide/sdh/tenants.md

```diff
@@ -0,0 +1,78 @@
+# Tenants
+
+This section allows you to add new [tenants](#tenants) and map environments to these [installations](./README.md#installations) to ensure updates [rollout](#rollout) correctly. 
+
+## Adding Tenant
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add tenants.
+{% endhint %}
+
+This involves the creation of new organizations where you wish to deploy s/w updates. Whenever you are onboarding a new client, you add them as a tenant. 
+
+1. Click **+ Add Tenant**.
+
+    ![Figure 1: Adding a Tenant](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/tenants-tab.jpg)
+
+2. Enter a name in **Tenant display name** field, e.g., `flareup.xyz`
+
+3. Add a unique identifier to your tenant in **Tenant ID** field, e.g., `flareup123` 
+
+4. (Optional) Add a description of the tenant.
```

Comment:
Word limit: 300 characters I guess

Reply:
Tested. It's 350.
The user will likely not benefit it by reading it from the doc since this field validation will be shown on the UI if the user exceeds the limit. 
<img width="1441" alt="image" src="https://github.com/user-attachments/assets/2c098b63-bcaf-421a-ad6a-4543ef60bf1a">